custom/lib_portal/models/issue_book.py

```diff
@@ -0,0 +1,44 @@
+
+from odoo import fields, models
+from datetime import datetime, timedelta
+
+class IssueBook(models.Model):
+    _name = "issue.book"
+    _description = "Library Book Lending Mangement"
+
+
+    fromDate = fields.Date('Start Date',default=datetime.today())
+    toDate = fields.Date(string="Due Date",default=datetime.today()+timedelta(days=30))
+
+    partner_id = fields.Many2one(
+        'res.partner', string='Customer', readonly=True,
+        required=True, change_default=True, index=True, tracking=1)
+    customer_name = fields.Char(related='partner_id.name')
+    customer_email = fields.Char('Email', related='partner_id.email')
+    book_id = fields.Many2one('product.template')
+    authorName = fields.Text('Author',related='book_id.authorName')
+    book_name = fields.Char(related='book_id.name')
+    emp_id = fields.Many2one('hr.employee', string='Employee')
```

Comment:
* You are using two different ways to set the label for these fields, both are accepted but I will recommend using only one way to do it. I prefer the 'string=' way because is more explicit.

Reply:
Noted.Changed all the labels to string = 'label'